### PR TITLE
bill-kunze adds Hinckley file

### LINKED ISSUE #12: bill-kunze practices using GitHub Flow
- [x] Create a branch off of `master`
- [x] Add some commits
- [x] Open a pull request
- [x] Collaborate 🌠 
- [x] Merge 😰 when we get at least 1 approval

### PR DESCRIPTION
**Pull request by bill-kunze**
The purpose of this Pull Request is to learn how to use Git / GitHub / new code management.  Simply adding Hinckley, MN file to the hometown files.

This PR closes #12

### CODE REVIEW

File: bill-kunze-hinckley.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#  Hinckley, MN
+
+### Restaurants to Visit
+- Tobies
+- Cassidy's
+- Dairy Queen
+- Taco Bell
+- Hardee's
```
Comment (jeffrey-jakubowsky) [inline bill-kunze-hinckley.md]: What no McDonalds?  I love McDonalds.,  Add that if they city has it.

**Comment by jeffrey-jakubowsky** [comment]
How many schools are in Hinckley and what are their names?

File: bill-kunze-hinckley.md (hunk 1 above)
Comment (bill-kunze) [reply bill-kunze-hinckley.md]: Oh there is now, but not when I was growing up.

**Comment by bill-kunze** [comment]
@jeffrey-jakubowsky I made some food and school changes for you.

**Review by jeffrey-jakubowsky — APPROVED** [review]
Looks great.  Happy to hear they have a McDonald's

**Pull request merged**